docs/guides/chip_tool_guide.md

```diff
@@ -0,0 +1,769 @@
+device side and may be different for your accessory.
+
+### Step 1: Prepare the Matter accessory.
+
+This tutorial is using the
+[Matter lighting-app example](https://github.com/project-chip/connectedhomeip/tree/master/examples/lighting-app)
+with the Bluetooth LE commissioning. However, you can adapt this procedure to
+other available Matter examples.
+
+Build and program the device with the Matter accessory firmware by following the
+example's documentation.
+
+### Step 2: Enable Bluetooth LE advertising on Matter accessory device.
+
+Some examples are configured to advertise automatically on boot. Other examples
+require physical trigger, for example pushing a button. Follow the documentation
+of the Matter accessory example to learn how Bluetooth LE advertising is enabled
+for the given example.
+
+### Step 3: Determine network pairing credentials
+
+You must provide the controller with network credentials that will be further
+used in the device commissioning procedure to configure the device with a
+network interface, such as Thread or Wi-Fi.
+
+#### Thread network credentials
+
+Fetch and store the current Active Operational Dataset from the Thread Border
+Router. Depending on if the Thread Border Router is running on Docker or
+natively on Raspberry Pi, execute the following commands:
+
+-   For Docker:
+
+    ```
+    sudo docker exec -it otbr sh -c "sudo ot-ctl dataset active -x"
+    0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8
+    Done
+    ```
+
+-   For native installation:
+
+    ```
+    sudo ot-ctl dataset active -x
+    0e080000000000010000000300001335060004001fffe002084fe76e9a8b5edaf50708fde46f999f0698e20510d47f5027a414ffeebaefa92285cc84fa030f4f70656e5468726561642d653439630102e49c0410b92f8c7fbb4f9f3e08492ee3915fbd2f0c0402a0fff8
+    Done
+    ```
+
+#### Wi-Fi network credentials
+
+The following Wi-Fi network credentials must be determined to commission the
+Matter accessory device to the Wi-Fi network in next steps:
+
+-   Wi-Fi SSID
+-   Wi-Fi password
+
+Matter specification does not define how the Thread or Wi-Fi credentials are
+obtained by Controller.
+
+For example, for Thread, instead of fetching datasets directly from the Thread
+Border Router, you might also use a different out-of-band method.
+
+For Wi-Fi, the steps required to determine the SSID and password may vary
+depending on the setup. For instance, you might need to contact your local Wi-Fi
+network administrator.
+
+### Step 4: Determine Matter accessory device's _discriminator_ and _setup PIN code_
+
+The controller uses a 12-bit value called _discriminator_ to discern between
+multiple commissionable device advertisements, as well as a 27-bit _setup PIN
+code_ to authenticate the device. You can find these values in the logging
+terminal of the device (for instance, UART). For example:
```

Comment:
This looks like a background information about CHIP Tool? If so, it should be described in the "application flow overview" I mentioned earlier...

Reply:
It's rather Matter specific. I would just replace "The controller" with  "The Matter", @greg-fer what do you think?